runtime/include/atomics/standard/chpl-atomics.h

```diff
@@ -0,0 +1,301 @@
+/*
+ * Copyright 2004-2016 Cray Inc.
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef _chpl_atomics_h_
+#define _chpl_atomics_h_
+
+#include "chpltypes.h"
+#include <assert.h>
```

Comment:
Good catch.  The include of assert.h has now been removed, and the result tested to make sure it really didn't need assert.h.